google/cloud/storage/internal/storage_stub_factory.cc

```diff
@@ -34,8 +34,27 @@ std::shared_ptr<grpc::Channel> CreateGrpcChannel(
     google::cloud::internal::GrpcAuthenticationStrategy& auth,
     Options const& options, int channel_id) {
   auto args = internal::MakeChannelArguments(options);
-  args.SetInt("grpc.channel_id", channel_id);
-  args.SetInt("grpc.use_local_subchannel_pool", 1);
+  // Use a local subchannel pool to avoid contention in gRPC
+  args.SetInt(GRPC_ARG_USE_LOCAL_SUBCHANNEL_POOL, 1);
+  // Use separate sockets for each channel, this is redundant since we also set
+  // `GRPC_ARG_USE_LOCAL_SUBCHANNEL_POOL`, but it is harmless.
+  args.SetInt(GRPC_ARG_CHANNEL_ID, channel_id);
+
+  // Disable queries see b/243676671, this is harmless as we do not use any
```

Comment:
"see b/243676671" isn't particularly useful for external code readers.  Perhaps substitute a small description, or at least make it parenthetical?

Reply:
reworded.